src/components/screens/Tour.js

```diff
@@ -30,6 +30,17 @@ export const Tour = ({ data }) => {
     title,
     webUrls
   } = data;
+  const rootRouteName = navigation.getParam('rootRouteName', '');
+  // action to open source urls
+  const openWebScreen = (webUrl) =>
+    navigation.navigate({
+      routeName: 'Web',
+      params: {
+        title: 'Tour',
+        webUrl,
+        rootRouteName
+      }
```

Comment:
why link it's not needed here?

Reply:
it is needed. the `webUrl` is the link (string with url of some web page).